rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/server/RMServiceImpl.java

```diff
@@ -474,6 +492,27 @@ private String executeFunctionReturnStreamAsString(Function<RestClient, InputStr
         }
     }
 
+    private String executeFunctionReturnStreamAsString(Function<RestClient, InputStream> function)
+            throws ServiceException, RestServerException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)